app/controllers/ProjectSidewalkAPIController.scala

```diff
@@ -862,10 +865,14 @@ class ProjectSidewalkAPIController @Inject()(implicit val env: Environment[User,
         writer.println(s"$labType Disagreed Count,${accStats.nDisagree}")
         writer.println(s"$labType Accuracy,${accStats.accuracy.map(_.toString).getOrElse("NA")}")
       }
+
+      writer.println(s"Launch Date, ${launchDate}")
       writer.close()
       Future.successful(Ok.sendFile(content = sidewalkStatsFile, onClose = () => sidewalkStatsFile.delete()))
     } else { // In JSON format.
-      Future.successful(Ok(APIFormats.projectSidewalkStatsToJson(LabelTable.getOverallStatsForAPI(filterLowQuality))))
+      Future.successful(Ok(APIFormats.projectSidewalkStatsToJson(LabelTable.getOverallStatsForAPI(filterLowQuality)) +
```

Comment:
Instead of having this as an additional piece of metadata that's separate from the `ProjectSidewalkStats` object, let's include it right in that case class! And I feel like it works best to have it as the first piece of data in the class instead of the last!

In order to make the changes as small as possible, what you can do is to essentially pass the launch date into the SQL query. The first line of the query would probably now look like
```
SELECT $launchDate AS launch_date
```

Reply:
Using this SQL query gives this response for the launch date:

![image](https://github.com/ProjectSidewalk/SidewalkWebpage/assets/77756028/1e2e53bc-a328-4f32-b6fb-5c02010221fa)

when it should display "2020-07-30". To be clear, is this SQL query supposed to just inject the `launchDate` variable into the query output? Or is the launch date actually stored in the database?

Reply:
It's not stored in the database in this scenario, you're just passing along a string!